components/public-api-server/pkg/apiv1/identityprovider.go

```diff
@@ -88,6 +88,8 @@ func (srv *IdentityProviderService) GetIDToken(ctx context.Context, req *connect
 	userInfo := oidc.NewUserInfo()
 	userInfo.SetName(user.Name)
 	userInfo.SetSubject(subject)
+	userInfo.AppendClaims("org_id", workspace.Workspace.OrganizationId)
```

Comment:
It's probs okay. If we need to change, we need to make it backwards compatible, however.

Reply:
I think we should go with snake case as it looks like the standard for OIDC JWTs: https://openid.net/specs/openid-connect-core-1_0.html#Claims

Reply:
So, the full name, e.g. `organization_id` @filiptronicek ? I would be happy with that here.

I presume US spelling as that's our typical default.

Reply:
Sorry for the confusion, @loujaybee! I meant shortening `org_id` to just `org`. I actually prefer `org`, because it's not prone to typos and has universal spelling.

`org_id` is alright with me.